doc/WebSite/Entity-History.md

```diff
@@ -113,9 +113,13 @@ While you can select tracked entities by configuration, you can use the
 ```
 
 All properties of MyEntity are tracked except MyProperty2 since it's
-explicitly disabled. The Audited attribute can be used to
+explicitly disabled. The **Audited** attribute can be used to
 save change logs for a desired property.
 
+When using **Audited** attribute on **entity** class, **EntityChange will be created** for entity in *Added/Modified/Deleted* state, regardless of whether any **PropertyChange** is being created.
+
+Similarly, when using **Audited** attribute on a **property**, **PropertyChange will be created**, regardless of whether there is any differences between the **new and old values** of the **property**.
```

Comment:
Still not right.

> As mentioned in https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5126#discussion_r357987933, this behaviour is neither consistent nor correct.
> ...
> Instead, we should fix the regression from #4870.

> - `PropertyChange` should only be created if **property** is changed.

```suggestion
When using **Audited** attribute on a **property**, **PropertyChange will be created** for property if there is any difference between the **new and old values** of the **property**.
```

Reply:
@acjh is this the only change needed ? Or does it still need extra work ?

Reply:
More work (fix the regression from #4870) is needed to resolve #5292.
Making this change without fixing the regression would make the documentation inconsistent with the behavior.